src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/autocomplete.ts

```diff
@@ -144,6 +145,13 @@ export async function suggest(
       return [...sourceCommandsSuggestions, ...recommendedQueriesSuggestions];
     }
 
+    const lastCommand = root.commands[root.commands.length - 1];
+    const lastCommandDefinition = getCommandDefinition(lastCommand.name);
+
+    if (lastCommandDefinition.commandsSuggestionsAfter) {
```

Comment:
Check my other comment. Let's change the `commandsSuggestionsAfter ` to only return the extra suggestions. So you won't need to reassign here, just push to the existing suggestions. I think it is cleaner